statistics-dashboard-common/src/main/java/eu/europeana/statistics/dashboard/common/api/request/StatisticsNonBreakdownValueFilter.java

```diff
@@ -0,0 +1,40 @@
+package eu.europeana.statistics.dashboard.common.api.request;
+
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.springframework.util.CollectionUtils;
+
+/**
+ * Similarly to filter type {@link StatisticsValueFilter} but this class does not contain breakdowns
+ */
+
+@JsonSerialize
+public class StatisticsNonBreakdownValueFilter implements StatisticsFilter{
```

Comment:
Hmm, I'm thinking. Could we make the hierarchy more streamlined? I mean:
StatisticsFilter -> StatisticsValueFilter(contains only values) -> StatisticsBreakdownValueFilter(now this inherits values and contains breakdown)
Not sure if it's possible in regards to (de)serialization, didn't try it..